assets/src/scripts/interactive/pages/review-request.jsx

```diff
@@ -135,6 +135,40 @@ function ReviewRequest() {
 
       {error ? <InputError>{error}</InputError> : null}
 
+      <section className="prose prose-blue mt-8 pt-6 border-t max-w-full">
+        <h2 className="sr-only">Read and agree</h2>
+        <ul>
+          <li>
+            Your analyses will be run on patient records for approximately 44%
```

Comment:
Should we be making it clear here that it's "primary care patient records"? We haven't really explained what the dataset is anywhere, yet, so might be worth tackling in another ticket.

Reply:
Good point, maybe also on the Interactive About page.